src/bindings/python/src/openvino/runtime/opset14/ops.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Factory functions for ops added to openvino opset14."""
+from functools import partial
+from typing import Literal, Optional, Union
+import logging
+
+import numpy as np
```

Comment:
I do not see numpy usage? is it really should be imported?

Reply:
I've kept it as it is present in every other opset file, but I can remove it in a followup PR with micro fixes